extension-config.json

```diff
@@ -127,7 +127,21 @@
     "accordionItems": {
       "type": "admin",
       "destination": "frontend",
-      "default": [],
+      "default": [
+        {
```

Comment:
Please reset the default value for the "accordionItems" setting. This is the configuration for AKF that's mentioned within the ticket.